### DISCUSSION TITLE
specific  long table format

### DISCUSSION THREAD

**Discussion by HeshamAMH** [lvjr/tabularray#295] [Q&A]
hi, thanks for your efforts. I am new in latex please can you share with me a code  for a long table (longtblr)  with the following format for caption.
![image](https://user-images.githubusercontent.com/35412852/185248920-0755188a-cf53-467b-9551-39ac18415527.png)

Thanks in advance.

**Comment by lvjr** [comment]
*Since this is a pure question, I have moved it from GitHub Issues to GitHub Discussions here.*

```latex
\documentclass{article}
\usepackage{tabularray}
\DefTblrTemplate{caption}{default}{%
  \textbf{\ExpTblrTemplate{caption-tag}{default}}\newline
  \ExpTblrTemplate{caption-text}{default}%
}
\DefTblrTemplate{capcont}{default}{%
  \textbf{\ExpTblrTemplate{caption-tag}{default}} (Continued)\newline
  \ExpTblrTemplate{caption-text}{default}%
}
\begin{document}
\begin{longtblr}[
  caption = Some long caption text,
  label = mytable
]{hlines}
  Alpha   & Beta  & Gamma   & Delta   \\
  Epsilon & Zeta  & Eta     & Theta   \\\pagebreak
  Iota    & Kappa & Lambda  & Mu      \\
  Nu      & Xi    & Omicron & Pi      \\
\end{longtblr}
\end{document}
```

PS: please try to provide a minimal example when asking questions.
